webpack/assets/javascripts/foreman_users.js

```diff
@@ -1,5 +1,6 @@
 import $ from 'jquery';
 import _ from 'lodash';
+import { notify } from './foreman_toast_notifications';
```

Comment:
I dont think this is enough as the function signature has changed.

Reply:
it will just pass an undef to the missing arguments which i think is fine (this is the same as we passed before in the deprecated function)

Reply:
or actually looking at it now i think you may be right